src/components/SearchBarView.tsx

```diff
@@ -1,84 +1,91 @@
-import { gql, useLazyQuery } from '@apollo/client'
-import React, { useEffect, useState } from 'react'
-import { SafeAreaView, StyleSheet, View } from 'react-native'
-import { Icon, Input } from 'react-native-elements'
+import { gql, useLazyQuery } from "@apollo/client";
+import React, { useEffect, useState } from "react";
+import { SafeAreaView, StyleSheet, View } from "react-native";
+import { Icon, Input } from "react-native-elements";
 
 // GraphQL fragment
-const COMPANY_TILE_DATA = gql` {
-    fragment CompanyTile on Business{
-        id
-        name
-        profilePicture
-        sustainabilityScore
-        customerScore
+const COMPANY_TILE_DATA = gql`
+  {
+    fragment
+    CompanyTile
+    on
+    Business {
+      id
+      name
+      profilePicture
+      sustainabilityScore
+      customerScore
     }
-}
-`
+  }
+`;
```

Comment:
This defines a fragment, not a query. I don't believe this will return a result.